config/routes.rb

```diff
@@ -656,7 +656,10 @@
     get '/' => 'start_page#show'
 
     get '/accessibility', to: 'content#accessibility'
-    get '/privacy-policy', to: 'content#privacy_policy', as: :privacy_policy
+    get '/privacy-policy', to: redirect('/privacy')
```

Comment:
could you add the full path for redirection `redirect('provider/privacy')` as redirection will not respect the route here

Reply:
sorted, my bad!